creator-node/src/config.js

```diff
@@ -601,12 +601,23 @@ const config = convict({
     env: 'saveFileForMultihashToFSIPFSFallback',
     default: true
   },
-  enableAsyncIPFSAdd: {
-    doc: 'Boolean indicating if `saveFileToIPFSFromFS()` should asynchronously add to ipfs and only use the content addressing logic',
+  enableIPFSAddTracks: {
+    doc: '[TRACKS] Boolean indicating if the CN should add content to the ipfs daemon, or rely only on content addressing logic',
```

Comment:
this is super dumb but could you specify like `true` to add to ipfs daemon, `false` to use content addressing?